pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -218,3 +221,32 @@ type NodeAffinityArgs struct {
 	// a specific Node (such as Daemonset Pods) might remain unschedulable.
 	AddedAffinity *v1.NodeAffinity
 }
+
+type StrategyType string
+
+const (
+	LeastAllocated           StrategyType = "LeastAllocated"
+	MostAllocated            StrategyType = "MostAllocated"
+	RequestedToCapacityRatio StrategyType = "RequestedToCapacityRatio"
+)
+
+// ScoringStrategy define StrategyType for node resource plugin
+type ScoringStrategy struct {
+	// Strategy selects which strategy to run.
+	Strategy StrategyType
```

Comment:
IMO, `Strategy` looks like more clearly.

Reply:
I think we should rename the type to `ScoringStrategyType` and the field name here should be just Type as Wei suggested (so it will be ScoringStrategy.Type)